src/styles.css

```diff
@@ -8,12 +8,20 @@
     @apply text-white text-left font-black text-4xl md:text-black sm:text-5xl lg:text-6xl md:w-2/3 w-full transition-all;
   }
 
+  h2 {
+    @apply font-black text-4xl sm:text-5xl lg:text-6xl transition-all ease-in-out;
+  }
+
   app-hero p {
     @apply bg-black bg-opacity-60 font-medium p-4 text-white text-base rounded-md w-full sm:text-lg md:w-2/3 lg:w-1/2 transition-all;
   }
 }
 
 @layer components {
+  .frame {
```

Comment:
This should not be here as it relates to a single component and not a general style